lms/resources/_js_config.py

```diff
@@ -46,7 +46,7 @@ def add_canvas_file_id(self, canvas_file_id):
         self._config["api"]["viaUrl"] = {
             "authUrl": self._request.route_url("canvas_api.oauth.authorize"),
             "path": self._request.route_path(
-                "canvas_api.files.via_url", file_id=canvas_file_id
+                "canvas_api.files.via_url", course_id=course_id, file_id=canvas_file_id
```

Comment:
The `via_url` API now includes the course ID in the path. This changes thepath that we pass to the frontend, so the frontend will not use the new path when calling the API

Reply:
I presume you meant "_will_ use the new path when calling the API"?

Reply:
Yep